libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/factory/RouterFactory.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.base.internal.factory
+
+import com.mapbox.navigation.base.ExperimentalMapboxNavigationAPI
+import com.mapbox.navigation.base.route.NavigationRouterRefreshError
+import com.mapbox.navigation.base.route.RouterFailure
+
+@ExperimentalMapboxNavigationAPI
```

Comment:
since it's an `internal` package, why need `Experimental` annotation?

Reply:
Moved. This factory is here for someone that implements their own `NavigationRouter` and would need to deliver refresh errors.